content/en/docs/tasks/tools/included/kubectl_installs_macos.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "macOS kubectl install"
+description: "Content for the macOS tab in the main page."
+toc_hide: true
+---
```

Comment:
This also looks like it might be intended to work like a Hugo [headless bundle](https://gohugo.io/content-management/page-bundles/#headless-bundle).

Reply:
Updated in https://github.com/kubernetes/website/pull/26682/commits/62ffd9d9a4e0fa6bac949ee8dfcfe12130afb357